kubetest2-gce/deployer/common.go

```diff
@@ -176,6 +176,14 @@ func (d *deployer) buildEnv() []string {
 	// e.g. https://github.com/kubernetes/kubernetes/issues/99480
 	env = append(env, "KUBE_CONFIG_FILE=config-test.sh")
 
+	if len(d.NodeScopes) > 0 {
```

Comment:
Similar to the pattern used for 

	if d.RuntimeConfig != "" {
		env = append(env, fmt.Sprintf("KUBE_RUNTIME_CONFIG=%s", d.RuntimeConfig))
	}

Maybe we just want to keep the pattern.
```suggestion
	if d.NodeScopes) != "" {
```